framework/src/index.js

```diff
@@ -1,4 +1,5 @@
 const Application = require('./controller/application');
+const GenesisBlockDevnet = require('./modules/chain/defaults/genesis_block_devnet');
```

Comment:
Its an object not a constructor function so we use camel case `genesis*`

Reply:
Done.